src/test/java/iss/sirius/SiriusApplicationTests.java

```diff
@@ -6,8 +6,17 @@
 @SpringBootTest
 class SiriusApplicationTests {
 
+    public UserControllerTest userControllerTest;
+
```

Comment:
Merg testele asa?

Reply:
Credeam ca o problema reiese de acolo dar nu era si am uitat sa le schimb inapoi :(

Reply:
Testele nu merg inca, lucerez la ele in background si vor mai fi modificari. Momentan poti sa le ignori si te anunt eu cand totul e ok